src/site/xdoc/building/index.xml

```diff
@@ -54,10 +54,10 @@
 				<thead>
 					<tr>
 						<th id="os">OS</th>
-						<th id="os_x86_x64">x86 vs. x64</th>
+						<th id="os_x86_x64">Architecture</th>

```

Comment:
Introducing a new header name is fine, but I suggest renaming those `*_x86_x64` phrases to `*_arch` as well. Things are clearer this way.